release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1316,6 +1316,11 @@ In {product-title} 4.11, support for VMware vSphere 7.0 Update 1 or earlier is d
 ==== Support for ESXi 7.0 Update 1 or earlier is deprecated
 In {product-title} 4.11, support for VMware ESXi 7.0 Update 1 or earlier is deprecated. While EXSi 7.0 Update 1 or earlier remains fully supported, Red Hat recommends that you use ESXi 7.0 Update 2 or later.
 
+[id="ocp-4-11-deprecated-features-crio-parameters"]
+==== Support for `pidsLimit` and `logSizeMax` CRI-O parameters will be deprecated
+
+In {product-title} 4.11, support for the `pidsLimit` and `logSizeMax` CRI-O parameters is expected to be deprecated in a future release. As part of the deprecation, the number of PIDs that can be created in a container or pod is no longer limited. It is recommended to use the `PodPidsLimit` and the `ContainerLogMaxSize` fields in the `KubeletConfig` CR instead.
```

Comment:
> As part of the deprecation, the number of PIDs that can be created in a container or pod is no longer limited.

this is not true. While CRI-O is no longer limiting the number of PIDs, there is a KubeletConfig option which does, and https://github.com/openshift/machine-config-operator/pull/3085 applies this by default

Reply:
and I did double check: 527e2a5f56e354e9166e41475620059c71a4e6c2 made it into the release-4.11 branch